src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
+const FULL_HEALTH = 100;
+const BITE_DAMAGE = 50;
+
 class Animal {
-  // write your code here
+  i;
+  static alive = [];
+
+  constructor(name, health = FULL_HEALTH) {
+    this.name = name;
+    this.health = health;
+    this.hidden = false;
```

Comment:
[CODE REDUNDANCY] - The `hidden` property is initialized in the `Animal` class but should only be relevant for the `Herbivore` class. It's better to remove it from the `Animal` class constructor to avoid confusion and unnecessary property initialization for `Carnivore` instances.